client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -14,45 +16,69 @@ import { STORE_KEY } from '../../stores/onboard';
 import { Step, usePath } from '../../path';
 
 interface Props {
-	inputRef: React.RefObject< HTMLInputElement >;
+	isVisible: boolean;
 }
-const SiteTitle: React.FunctionComponent< Props > = ( { inputRef } ) => {
+
+const SiteTitle: React.FunctionComponent< Props > = ( { isVisible } ) => {
 	const { __: NO__ } = useI18n();
-	const { siteTitle } = useSelect( select => select( STORE_KEY ).getState() );
+	const { siteTitle, siteVertical } = useSelect( select => select( STORE_KEY ).getState() );
 	const { setSiteTitle } = useDispatch( STORE_KEY );
 	const history = useHistory();
 	const makePath = usePath();
+	const inputRef = React.useRef< HTMLSpanElement >( document.createElement( 'span' ) );
 
-	const handleChange = ( e: React.ChangeEvent< HTMLInputElement > ) =>
-		setSiteTitle( e.target.value.trim().length ? e.target.value : '' );
+	const handleKeyDown = ( e: React.KeyboardEvent< HTMLSpanElement > ) => {
+		if ( e.keyCode === ENTER ) {
+			// As last input on first step, hitting 'Enter' should direct to next step.
+			e.preventDefault();
+			history.push( makePath( Step.DesignSelection ) );
+		}
+	};
+
+	const handleKeyUp = ( e: React.KeyboardEvent< HTMLSpanElement > ) =>
+		setSiteTitle( e.currentTarget.innerText.trim().length ? e.currentTarget.innerText : '' );
+
+	React.useEffect( () => {
+		if ( siteTitle ) {
+			inputRef.current.innerText = siteTitle;
+		}
+	}, [] ); // eslint-disable-line react-hooks/exhaustive-deps
 
-	const value = siteTitle.length ? siteTitle : '';
+	React.useEffect( () => {
+		if ( siteVertical?.label ) {
+			inputRef.current.focus();
+		}
+	}, [ siteVertical, inputRef ] );
 
 	// translators: Form input for a site's title where "<Input />" is replaced by user input and must be preserved verbatim in translated string.
 	const madlibTemplate = NO__( 'Itʼs called <Input />' );
 	const madlib = __experimentalCreateInterpolateElement( madlibTemplate, {
 		Input: (
-			<input
+			<span
+				contentEditable
+				tabIndex={ 0 }
+				role="textbox"
+				aria-multiline="true"
+				spellCheck={ false }
```

Comment:
Grammarly is still jumping in.
![image](https://user-images.githubusercontent.com/17054134/78230368-86656400-74d1-11ea-949a-8762df176ed2.png)

This attrib seems to fix it (tested) 
```suggestion
				spellCheck={ false }
				data-gramm_editor="false"
```
Taken from [here](https://stackoverflow.com/a/46777787).

Reply:
> This has changed to data-gramm="false" now as I have been advised by Grammarly team.

However, this feels like a specific case. As @simison mentioned when seeing this for the first time, maybe in this case, to avoid typos, we might considering leaving it as it is. The browser `spellCheck` felt like a visual issue (double bottom border) but Grammarly icon isn't looking that bad. cc @dubielzyk

Reply:
This is an edge case and like you're saying, it's kinda working as intended. Let's leave it for now.